### PR TITLE
Add additional solution packaging parameters

### PR DESCRIPTION
**Pull request by paulbreuler**
Bump version of PP-CLI-Wrapper to x.49

Note that use of single quotes is a non-default option in VS Code 
image

Should potentially be enforced with a style check if single is preferred over double quotes.

### CODE REVIEW

**Review by tehcrashxor — APPROVED** [review]

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -65,12 +65,25 @@ jobs:
         user-name: ${{ env.WF_USERNAME }}
         password-secret: ${{ secrets.PASSWORD_PPDEVTOOLS }}
 
+    - name: Test pack-solution action
```
Comment (davidjenni) [inline .github/workflows/pull-request.yml:68]: nit: fix name to unpack

File: pack-solution/action.yml [hunk 2]
```diff
@@ -1,22 +1,59 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
-name: 'pack-solution'
-description: 'Power Platform Pack Solution'
+name: "pack-solution"
```
Comment (davidjenni) [inline pack-solution/action.yml:3]: why the churn to double quotes?

File: unpack-solution/action.yml [hunk 3]
```diff
@@ -1,27 +1,65 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
-name: 'unpack-solution'
-description: 'Power Platform Unpack Solution'
+name: "unpack-solution"
```
Comment (davidjenni) [inline unpack-solution/action.yml:3]: same as pack's action.yml: why the change to double quotes?

**Review by davidjenni — CHANGES_REQUESTED** [review]

File: pack-solution/action.yml (hunk 2 above)
Comment (paulbreuler) [reply pack-solution/action.yml:3]: Good question... looking the VS Code settings single quote is a non-default option. I updated mine to match the code. 
image

**Comment by davidjenni** [comment]
I prefer any drastic whitespace or quoting changes to be separate, non-functionality impact changes (i.e. anything that goes beyond a few opportunistic fixups), since it adds a lot of diff noise.
I know the existing .editorconfig VS Code extension doesn't have the ability to preset per-repo styling settings like quotes (we liberally use that in VS for C#), but I think there is a way to have per-repo VSCode settings we can checkin to better control consistency

**Review by davidjenni — APPROVED** [review]
I prefer any drastic whitespace or quoting changes to be separate, non-functionality impact changes (i.e. anything that goes beyond a few opportunistic fixups), since it adds a lot of diff noise.
I know the existing .editorconfig VS Code extension doesn't have the ability to preset per-repo styling settings like quotes (we liberally use that in VS for C#), but I think there is a way to have per-repo VSCode settings we can checkin to better control consistency
